src/bci_build/package/postfix/entrypoint/entrypoint.sles.sh

```diff
@@ -427,11 +418,5 @@ configure_postfix
 # before starting services
 rm -f /var/spool/postfix/pid/master.pid
 
-if [ "$1" = 'postfix' ]; then
-    start_daemons "$@"
-    echo "postfix running and ready"
-    echo "[info] refer to postfix manual pages at https://www.postfix.org/postfix-manuals.html"
-    sleep infinity & wait $!
-else
-    exec "$@"
-fi
+exec "$@"
+echo "[info] refer to postfix manual pages at https://www.postfix.org/postfix-manuals.html"
```

Comment:
This line will never be executed due to the exec before.

Reply:
I swapped the two statements.